.github/workflows/worker-deploy.yml

```diff
@@ -42,10 +42,14 @@ jobs:
             SUPABASE_URL
             SUPABASE_KEY
             KERNEL_PUBLIC_KEY
+            CLOUDFLARE_ACCOUNT_ID
```

Comment:
If these are not set in the environment, wouldn't it crash during the depoyment?

Reply:
The action won't deploy a worker without the account_id so yeah it would